earth_enterprise/src/maps/api/bootstrap.js

```diff
@@ -22,21 +22,28 @@ google.maps = google.maps || {};
     document.write('<script src="' + src +
                    '" type="text/javascript"></script>');
   }
-  
+
   GEE_API_PATH = '/maps/api/js';
   FUSION_API_PATH = '/maps/api';
 
   google.maps.Load = function(apiLoad) {
-    /* http://mt0.google.com/vt */    apiLoad([,[,,,],['en-US',,,,,,GEE_BASE_URL + '/maps/api/icons/',],[GEE_BASE_URL + GEE_API_PATH,'internal'],[0],0.0,,,,,], loadScriptTime);
+    /* http://mt0.google.com/vt */    apiLoad([,[,,,],['en-US',,,,,,GEE_BASE_URL + '/maps/api/icons/',
+                                                        GEE_BASE_URL /* GE metrics */,
+                                                        ,
+                                                        GEE_BASE_URL /* GE metrics */],
+                                                        [GEE_BASE_URL + GEE_API_PATH,'internal'],[0],0.0,,,,,], loadScriptTime);
   };
 
   var loadScriptTime = (new Date).getTime();
 
-  getScript(GEE_BASE_URL + '/js/jquery.js');
+  // Only load jQuery if it isn't loaded already.
+  if (!window.jQuery) {
```

Comment:
Do we need this change in the other versions of bootstrap.js?

Reply:
Update: none of the other files load jQuery, so this behavior is correct.

Reply:
I think the idea was not to change the behavior of the old versions.  The maps API seems to work fine with the newer jQuery, but it's possible people are depending on maps loading the older 1.5 version over the version they have loaded.

Now that I'm thinking about it, though, we should mention this in the release notes.